systems/subscriber/sim-pool/pkg/utils/utils.go

```diff
@@ -0,0 +1,75 @@
+package utils
+
+import (
+	"github.com/jszwec/csvutil"
+	pb "github.com/ukama/ukama/systems/subscriber/sim-pool/pb/gen"
+	"github.com/ukama/ukama/systems/subscriber/sim-pool/pkg/db"
+)
+
+func PoolStats(slice []db.Sim) *pb.GetStatsResponse {
+	total := len(slice)
+	failed := 0
```

Comment:
Not incerementing or checking fail state ?

Reply:
I have included a new field for tracking failed SIMs in the simPool model struct, and modified the function to take into account the failed SIMs while returning the stats.